static/app/views/settings/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -132,6 +139,31 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
     return this.metadata.features;
   }
 
+  renderTabs() {
+    // TODO: Convert to styled component
+    const {integrationSlug} = this.props.params;
```

Comment:
i think we normally use `provider.key` for this which should be available in this component